### DISCUSSION TITLE
K3s system upgrade through Rancher

### DISCUSSION THREAD

**Discussion by Ezinho031** [rancher/system-upgrade-controller#318] [Q&A]
Hi everyone!

Going through the K3s documentation about automatic upgrades I saw a warning section mentioning that the upgrades plans and stuff should be managed in the Rancher UI. But I'm not seeing any option, section or menu about this in my rancher installation. Is there something I'm missing?

![image](https://github.com/user-attachments/assets/77cbc3a9-0b7d-44c2-bb74-48c310cc2519)
![image](https://github.com/user-attachments/assets/a7184bfd-6b47-487c-a2a8-ddb292b93a1b)

**Comment by brandond** [comment]
In the Cluster Management view, on the imported cluster, there should be an option to select the Kubernetes version. When you change the version, SUC and a Plan will be deployed to manage the version.
